### PR TITLE
Possible fix for dropdown options overlapping

### PR DESCRIPTION
**Pull request by thomasdarvik**
Options overlapping when text breaks over two lines and the dropdown is small enough.
I did not find a better way of handling the `.select div` and `!important`, but I am open for suggestions of course.

This might not be the correct way of handling the problem, but it fixed my problem, so it might be a step in the right direction.

### CODE REVIEW

File: source/styles.css [hunk 1]
```diff
@@ -21,3 +22,6 @@
 .VirtualizedSelectSelectedOption {
   font-weight: bold;
 }
+.Select div {
+    box-sizing: content-box !important;
```
Comment (bvaughn) [inline source/styles.css]: I don't think I'm comfortable with this. In general, `!important` rules are kind of gross– and changing the `box-sizing` might break existing custom styles.

File: source/styles.css [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
   display: flex;
   -webkit-box-align: center;
-      -ms-flex-align: center;
-          align-items: center;
+  -ms-flex-align: center;
+  align-items: center;
   padding: 0 .5rem;
   cursor: pointer;
+  height: auto;
+  white-space: normal;
```
Comment (bvaughn) [inline source/styles.css]: This is the default value. Why specify it explicitly?

File: source/styles.css [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
   display: -ms-flexbox;
   display: flex;
   -webkit-box-align: center;
-      -ms-flex-align: center;
-          align-items: center;
+  -ms-flex-align: center;
+  align-items: center;
   padding: 0 .5rem;
   cursor: pointer;
+  height: auto;
```
Comment (bvaughn) [inline source/styles.css]: This is the default value. Why specify it explicitly?

Comment (bvaughn) [reply source/styles.css]: This should also be overridden by react-virtualized's inline `style` (which has a fixed height)

File: source/styles.css (hunk 1 above)
Comment (thomasdarvik) [reply source/styles.css]: Agree. Is there any way of implementing the `box-sizing` in a safer way?

Comment (bvaughn) [reply source/styles.css]: Leave it up to applications to specify their own styles. (I just don't think this should be in the library as a default)

**Pull request closed without merging**